client/app/cloud/project/compute/infrastructure/volume/addEdit/cloud-project-compute-infrastructure-volume-addEdit.controller.js

```diff
@@ -214,6 +217,18 @@ angular.module('managerApp')
         }
       };
 
+      // --------- available REGIONS panel ---------
+
+      self.getAvailableRegions = function getAvailableRegions() {
```

Comment:
This API call seems to be made at least three times, Maybe it could go into a `service` ?